cfgov/jinja2/v1/_includes/macros/subscribe.html

```diff
@@ -26,7 +26,7 @@
         <input id="email-subscribe-form_email"
                type="email"
                name="email"
-               placeholder="example@mail.com"
+               placeholder="mail@example.com"
```

Comment:
I'm not sure this macro i actually still used anywhere, but doesn't hurt to make the change anyway.